widget/info.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "infoGraphics",
+    "title": "Info Graphics",
```

Comment:
snapshots will be added here as well which will show edit and view panel screenshots